cypress/cypress.config.ts

```
@@ -1167,6 +1172,80 @@ export default defineConfig({
+              if (!groupActivity) {
+                throw new Error(`Live quiz ${activityName} not found`)
+              }
+
+              await prisma.groupActivity.update({
+                where: { id: groupActivity.id },
+                data: { status },
+              })
+            }
+          } finally {
+            await prisma.$disconnect()
+            return true
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)